tests/intern.js

```diff
@@ -18,6 +18,7 @@ function (args, topic, firefoxProfile) {
   var fxaContentRoot = args.fxaContentRoot || 'http://127.0.0.1:3030/';
   var fxaEmailRoot = args.fxaEmailRoot || 'http://127.0.0.1:9001';
   var fxaOauthApp = args.fxaOauthApp || 'http://127.0.0.1:8080/';
+  var fxaUntrustedOauthApp = args.fxaUntrustedOauthApp || 'http://127.0.0.1:10139/';
```

Comment:
We should add this arg to `"test-latest":` in package.json, pointing to https://321done-latest.dev.lcip.org/